tilavarauspalvelu/management/commands/data_creation/create_users.py

```diff
@@ -1,19 +1,17 @@
-# ruff: noqa: S311
-
 from django.contrib.auth.hashers import make_password
 
+from tests.factories import UserFactory
 from tilavarauspalvelu.enums import ReservationNotification, UserRoleChoice
 from tilavarauspalvelu.models import User
 
 from .utils import with_logs
 
 
-@with_logs()
+@with_logs
 def _create_users() -> list[User]:
-    # TODO: Refactor this function
     users: list[User] = [
         # Django admin user
-        User(
+        UserFactory.build(
```

Comment:
These factories need to be updated, since `preferred_language` is now a non-nullable field after #1380

`django.db.utils.IntegrityError: null value in column "preferred_language" of relation "user" violates not-null constraint`

Reply:
Fixed